### PR TITLE
MEX-459 "Confirmed" checkbox in Diagnosis wiget should instead say "First time" ("1a vez")

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/MEX-459

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/mexico/consult.xml [hunk 1]
```diff
@@ -1046,6 +1046,24 @@
         </h1>
     </ifMode>
 
+    <script type="text/javascript">
+        // Hack to make "Confirmado" appear as "1a vez"
+        // See https://pihemr.atlassian.net/browse/MEX-459
+
+        $('body').on('DOMNodeInserted', 'div', function () {
+            const targetText = "1a vez";
+            const confirmedLabel = jq('label:contains("Confirmado")');
```
Comment (mogoodrich) [inline omod/src/main/webapp/resources/htmlforms/mexico/consult.xml]: Is there an id on this label?  Seems like there's a risk here if someone changes the display test to something other than "Confirmado", which I would think someone might assume they could do without causing any downstream changes?

File: omod/src/main/webapp/resources/htmlforms/mexico/consult.xml [hunk 2]
```diff
@@ -1046,6 +1046,24 @@
...
+            const confirmedText = $(confirmedLabel[0])
+                    .contents()
+                    .filter(function () {
+                        return this.nodeType == 3;
+                    })[1];
```
Comment (mogoodrich) [inline omod/src/main/webapp/resources/htmlforms/mexico/consult.xml]: Not quite sure what this is doing, but I assume it works.

File: omod/src/main/webapp/resources/htmlforms/mexico/consult.xml [hunk 3]
```diff
@@ -1046,6 +1046,24 @@
         </h1>
     </ifMode>
 
+    <script type="text/javascript">
+        // Hack to make "Confirmado" appear as "1a vez"
+        // See https://pihemr.atlassian.net/browse/MEX-459
+
+        $('body').on('DOMNodeInserted', 'div', function () {
+            const targetText = "1a vez";
+            const confirmedLabel = jq(jq('input[data-ng-model="d.confirmed"]')[0]).parent();
```
Comment (brandones) [inline omod/src/main/webapp/resources/htmlforms/mexico/consult.xml]: @mogoodrich Good thought about the lookup by label. I've changed it to refer to a data property of the <input> tag it contains.

**Comment by brandones** [comment]
Thanks for the review @mogoodrich !

**Pull request merged**